src/Microsoft.Azure.WebJobs.ServiceBus/WebJobs.ServiceBus.csproj

```diff
@@ -6,6 +6,8 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyName>Microsoft.Azure.WebJobs.ServiceBus</AssemblyName>
+    <PackageId>Microsoft.Azure.WebJobs.Extensions.ServiceBus</PackageId>
```

Comment:
So both EventHubs and ServiceBus packages now have the right Extensions naming, while their namespace/assembly name remain Microsoft.Azure.WebJobs.EventHubs and Microsoft.Azure.WebJobs.ServiceBus.

As a nit - could you change the project names to WebJobs.Extensions.EventHubs and WebJobs.Extensions.ServiceBus to match WebJobs.Extensions.Storage?